extensions/aws/processors/PutS3Object.h

```diff
@@ -97,54 +111,46 @@ class PutS3Object : public S3Processor {
 
   class ReadCallback {
    public:
-    static constexpr uint64_t MAX_SIZE = 5_GiB;
-    static constexpr uint64_t BUFFER_SIZE = 4_KiB;
-
-    ReadCallback(uint64_t flow_size, const minifi::aws::s3::PutObjectRequestParameters& options, aws::s3::S3Wrapper& s3_wrapper)
-      : flow_size_(flow_size)
-      , options_(options)
-      , s3_wrapper_(s3_wrapper) {
+    ReadCallback(uint64_t flow_size, const minifi::aws::s3::PutObjectRequestParameters& options, aws::s3::S3Wrapper& s3_wrapper,
+          uint64_t multipart_threshold, uint64_t multipart_size, core::logging::Logger& logger)
```

Comment:
I think the class definition could be moved to the .cpp file, too. I don't think it's used anywhere outside this processor.

Reply:
Updated in 0464ed5fc57b119da618142c671f2b985b4ae127